components/brave_wallet_ui/image/image.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+const urlParam = new URLSearchParams(window.location.search).get('targetUrl')
+const imgUrl = urlParam?.replace('chrome://image?', '') || ''
```

Comment:
can you add some validation here to make sure imgUrl has a whitelisted protocol like `https://`? then i think we can dismiss the codeql alerts on  https://github.com/brave/brave-core/pull/14535/files#diff-dd073161cfe75a2be24e73c54b34cb15208cf039e92d2984f566e1d04c3618f9R22

Reply:
or better yet, make sure `imgUrl` has the expected origin https://github.com/brave/brave-core/pull/14535/files#r941851887